pkg/eventprocessor/impl/executor/impl/blockscope.go

```diff
@@ -60,10 +60,10 @@ func newBlockScope(
 //
 // If the events execution is successful, it returns the result.
 // If the events execution isn't successful, we have two cases:
-// 1) If caused by controlled error (e.g: invalid SQL), it will return a (res, nil) where
-//    res contains the error message.
-// 2) If caused by uncontrolled error (e.g: can't access the DB), it returns ({}, err). The caller should retry
-//    executing this transaction events when the underlying infrastructure problem is solved.
+//  1. If caused by controlled error (e.g: invalid SQL), it will return a (res, nil) where
+//     res contains the error message.
+//  2. If caused by uncontrolled error (e.g: can't access the DB), it returns ({}, err). The caller should retry
```

Comment:
Was this surfaced by some lint? (Agree with the change, just trying to force these things as `make lint` if possible so we have this as an automatic rule)

Reply:
Yea, somehow my vscode go linter or gofumpt formatting did this. Will try to get our `make lint` flagging these things too, not sure why it's not. Pretty sure this was done by gofumpt on file save in vscode.

Reply:
Interesting. This was fixed because of my auto format on save with the formatter set to gofumpt. But, if i revert this change using vi (no format on save) and run `make lint`, it does fail linting:

```
> make lint
go run github.com/golangci/golangci-lint/cmd/golangci-lint@v1.46.2 run
pkg/eventprocessor/impl/executor/impl/blockscope.go:63: File is not `gofumpt`-ed (gofumpt)
//  1) If caused by controlled error (e.g: invalid SQL), it will return a (res, nil) where
pkg/eventprocessor/impl/executor/impl/blockscope.go:65: File is not `gofumpt`-ed (gofumpt)
//  2) If caused by uncontrolled error (e.g: can't access the DB), it returns ({}, err). The caller should retry
exit status 1
make: *** [lint] Error 1
```

I'm running Go 1.19, could that have something to do with it? I can't think of how else this wouldn't fail in CI and on your machine.

Reply:
@asutula , I confirmed that's the case. With go1.17.2, the lint doesn't fail, but if I update to go1.19 it does.
Maybe in go.1.19 something changed in the stdlib reg parsing files and the AST changed, thus gofumpt can detect something new (without it's code being changed).

We should change in `.github/workflows/review.yaml` the version from go1.17 to go1.19. We're actually using go1.19 for tests and Dockerfiles.

Reply:
Ok nice, will do that. I also noticed there is an updated version of golangci-lint that detects a few new issues about missing package comments. Will look at updating that as well.

Reply:
The package comment requirements coming from `golangci-lint` `revive` plugin when upgrading to the latest `golangci-lint` are a little confusing to me at the moment. Will leave `golangci-lint` at the version we're already using and just update the go version. Can revisit that upgrade later.